src/Core/Worker.php

```diff
@@ -211,6 +211,34 @@ public static function execute($queue) {
 		// Check for existance and validity of the include file
 		$include = $argv[0];
 
+		if (method_exists(sprintf('Friendica\Worker\%s', $include), 'execute')) {
```

Comment:
Why not move this condition on line 257 with `if (function_exists($funcname)) {` and then discriminate on the existing `self::execFunction($queue, $funcname, $argv, false);` instead of duplicating so much code?

Reply:
The duplicated code will be removed when the last files had been converted.

Reply:
When do you expect to be done with the conversion? Especially if it's temporary, we might as well modify the least amount of code.